api/config/config_preprod.json

```diff
@@ -45,15 +45,17 @@
     "operationalTerraWorkspaceNamespace": "aouwgscohortextraction-preprod",
     "operationalTerraWorkspaceName": "aouwgscohortextractionpreprod",
     "operationalTerraWorkspaceBucket": "fc-5d8e6afe-9429-44ae-a22a-67b09705d1b4",
-    "extractionMethodConfigurationNamespace": "aouwgscohortextraction-preprod",
-    "extractionMethodConfigurationName": "GvsExtractCohortFromSampleNames",
-    "extractionMethodConfigurationVersion": 5,
-    "extractionMethodLogicalVersion": 3,
     "extractionDestinationDataset": "fc-aou-cdr-preprod-ct.wgs_extraction_destination",
-    "minExtractionScatterTasks": 350,
-    "extractionScatterTasksPerSample": 4,
     "gatkJarUri": "gs:\/\/all-of-us-workbench-test-genomics\/wgs\/gatk-package-4.2.0.0-533-g9c2aa67-SNAPSHOT-local.jar",
-    "enableJiraTicketingOnFailure": true
+    "enableJiraTicketingOnFailure": true,
+    "cdrv7": {
```

Comment:
I saw you use v7 everywhere, please note that this is not just v7, it's all versions prior 8..

Reply:
Yes.  I have added comments elsewhere about this.

I couldn't think of a good name for this structure which expresses that.  Any ideas?

Reply:
not sure if `legacy` good name or not. Don't have other better ideas

Reply:
sgtm - renamed to legacy